g2p/cli.py

```diff
@@ -7,7 +7,7 @@
 from networkx import draw, has_path
 
 from g2p.transducer import CompositeTransducer, Transducer
-from g2p.mappings.create_fallback_mapping import align_to_dummy_fallback
+from g2p.mappings.create_fallback_mapping import align_to_dummy_fallback, dummy_inventory
```

Comment:
```suggestion
from g2p.mappings.create_fallback_mapping import align_to_dummy_fallback, DUMMY_INVENTORY
```

Reply:
I like that you pulled this out of the function, but let's [capitalize](https://www.python.org/dev/peps/pep-0008/#constants) it